### PR TITLE
Enable circuit breaker

### PR DESCRIPTION
**Pull request by jkempster34**
### Trello card
https://trello.com/c/e9vBMLJE

### Context
Circuit breaker middleware has been added to the API Client Gem. It is enabled via the following configuration:
```rb
GetIntoTeachingApiClient.configure do |config|
  config.circuit_breaker = {
    enabled: true, # Enables the circuit breaker middleware
    threshold: 5, # Number of failures that will cause the circuit breaker to trip
    timeout: 5.minutes # Amount of time until the circuit breaker will attempt to recover
  }
end
```

### Changes proposed in this pull request
- Enable circuit breaker

### CODE REVIEW

File: config/initializers/git_api_client.rb [hunk 1]
```diff
@@ -14,4 +14,10 @@
   end
 
   config.cache_store = Rails.cache
+
+  config.circuit_breaker = {
+    enabled: true,
+    threshold: 5,
```
Comment (ethax-ross) [inline config/initializers/git_api_client.rb:20]: What was the rational behind the threshold? I can't remember if we talked about this or not?

Comment (jkempster34) [reply config/initializers/git_api_client.rb:20]: Number of consecutive errors that will break the circuit

Comment (ethax-ross) [reply config/initializers/git_api_client.rb:20]: Yeah sorry I mean where did we get 5 min 5 minutes from?

**Review by ethax-ross — APPROVED** [review]
LGTM - I think its worth logging to Sentry when a circuit breaks though so we have a clear indication for when users are going to see a change/service unavailable message. It should just be a case of calling `capture_message` here:

https://github.com/DFE-Digital/get-into-teaching-app/blob/f392f67121a0e8814d66e0de738847ad42921740/app/controllers/concerns/circuit_breaker.rb#L7

**Pull request merged**